reference/tax_provider.yml

```diff
@@ -340,6 +340,8 @@ paths:
       summary: Commit Tax Quote
       description: |-
         Submit the quote request to be persisted by the enabled third-party tax provider. A commit operation is intended to be submitted once only, when the Order has been confirmed and paid.
+        
+        Merchants may adjust when commit operations occur by adjusting the document submission strategy in their store tax settings. The selected document submission strategy will adjust whether order status or payment status is used to determine whether the order is paid. For more information, see the [Tax Settings API Reference](/docs/rest-management/tax-settings).
```

Comment:
`used to determine if the order is paid`?
Therefore also escaping the double whether.
But not a biggy.

Reply:
Oh no, another double up! Thanks.